README.md

```diff
@@ -12,51 +12,59 @@ There is a public calendar if you would like to subscribe to these events:
 
 ## Next meeting
 
-- 23 October 2024 ([Meeting Agenda](2024-10-23/agenda.md))
-- Content Meeting:
-  - 28 October 2024
+- 27 February 2025: ([Meeting Agenda](2025-02-27/agenda.md)) on the [MDN Discord](https://developer.mozilla.org/discord)
+- Content Meeting: 3rd March 2025
 
 ## Previous meetings
 
-- [25 September 2024](2024-09-25)
-  - ([Agenda](2024-09-25/agenda.md))
-  - ([Minutes](2024-09-25/minutes.md))
-- [02 September 2024 - Content Meeting](2024-09-02)
-  - ([Agenda](2024-09-02/agenda.md))
-  - ([Minutes](2024-09-02/minutes.md))
-- [21 August 2024](2024-08-21)
-  - ([Agenda](2024-08-21/agenda.md))
-  - ([Minutes](2024-08-21/minutes.md))
-- [23 July 2024](2024-07-23)
-  - ([Agenda](2024-07-23/agenda.md))
-  - ([Minutes](2024-07-23/minutes.md))
-- [24 June 2024](2024-06-24)
-  - [Agenda](2024-06-24/agenda.md)
-  - [Minutes](2024-06-24/minutes.md)
-- [29th May 2024](2024-05-29)
-  - [Agenda](2024-05-29/agenda.md)
-  - [Minutes](2024-05-29/minutes.md)
+- [23 October 2024](2024-10-23)
+  - ([Agenda](2024-10-23/agenda.md))
+  - ([Minutes](2024-10-23/minutes.md))
+- [25 September 2024](2024/2024-09-25)
+  - ([Agenda](2024/2024-09-25/agenda.md))
+  - ([Minutes](2024/2024-09-25/minutes.md))
+- [02 September 2024 - Content Meeting](2024/2024-09-02)
+  - ([Agenda](2024/2024-09-02/agenda.md))
+  - ([Minutes](2024/2024-09-02/minutes.md))
```

Comment:
Is this right?

```suggestion
- [25 September 2024 - Content Meeting](2024/2024-09-25)
  - ([Agenda](2024/2024-09-25/agenda.md))
  - ([Minutes](2024/2024-09-25/minutes.md))
```

Reply:
Yeah - that doesn't add up. We moved it to Google Docs later, so we're not maintaining the content meeting here. I'll edit!